gms/feature_service.hh

```diff
@@ -58,6 +60,9 @@ class feature_service final : public peering_sharded_service<feature_service> {
     std::unordered_map<sstring, std::reference_wrapper<feature>> _registered_features;
 
     feature_config _config;
+
+    shared_ptr<persistent_feature_enabler> _enabler;
```

Comment:
Please, no. The persistent_feature_enabler already references feature_service, this back-link is asking for troubles. But this comment is connected to my other one regarding the controversial feature_service API change.